README.md

```diff
@@ -16,29 +16,32 @@ This project aims to build the official site for the community page so that any
 
 ## Tech Stack 🗃
 
- <img src="https://img.shields.io/badge/-MongoDB-yellow?style=flat&logo=mongoDB"> <img src="https://img.shields.io/badge/-expressJS-yellowgreen"> <img src="https://img.shields.io/badge/-ReactJS-%2300bfff%20">  <img src="https://img.shields.io/badge/-NodeJS-black?style=flat&logo=node.js">  
+ <img src="https://img.shields.io/badge/-MongoDB-yellow?style=flat&logo=mongoDB"> <img src="https://img.shields.io/badge/-expressJS-yellowgreen"> <img src="https://img.shields.io/badge/-ReactJS-%2300bfff%20">  <img src="https://img.shields.io/badge/-NodeJS-black?style=flat&logo=node.js"> 
+ 
+##  Getting Started 👨‍💻
+
+1. `git clone`
+2. `cd Community-Website`
 
 ## Frontend ⚛️
 
 Following are the steps to run the frontend of the community-website.All the code will go in `frontend` directory. 
 
-1. `git clone`
-2. Navigate to `Community-Website` folder using `cd Community-Website`
-3. `cd frontend`
-4. Run `npm install`. It will install all the required packages and dependencies.
-5. Run `npm start` to run the server.
-6. Navigate to [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+1. `cd frontend`
```

Comment:
Please be a little more descriptive like the backend section.

Reply:
but in the heading, it was mentioned...

Reply:
I didn't get your point. Please elaborate!

Reply:
Means every section has its own heading like 'frontend', 'backend', so how can I be more descriptive.